manifests/cpp.yml

```diff
@@ -211,6 +211,7 @@ tests/:
       Test_Parametric_Otel_Baggage: missing_feature (otel api is not supported)
       Test_Parametric_Otel_Current_Span: missing_feature (otel api is not supported)
       Test_Parametric_Otel_Trace_Flush: missing_feature (otel api is not supported)
+    test_process_discovery.py: v1.0.1.dev
```

Comment:
The error in the CI is not very clear 😭 

1. versions can only be set at class level (I'm not sure if it's a good limitation ... )
2. and keys must be sorted

I'll fix both issues